server/acl_casbin_policy_developer.csv

```diff
@@ -66,3 +66,4 @@ p, DEVELOPER, /vcs/{id}, GET
 p, DEVELOPER, /plan, GET
 p, DEVELOPER, /plan, PATCH
 p, DEVELOPER, /setting, GET
+p, DEVELOPER, /labelkey, GET
```

Comment:
Just call it label?

Reply:
how does the api for database label look like?

Technically, label key != label, however, from the user's perspective and our documentation purpose, it may just make it easy to call it workspace label unless we do have another actual concept called workspace label

Reply:
Okay, I'm convinced to use label here. Database label can come together with DB API in a declarative way following k8s/GCP API.